extprocconfig/extprocconfig.go

```diff
@@ -105,8 +105,19 @@ type Backend struct {
 	OutputSchema VersionedAPISchema `yaml:"outputSchema"`
 	// Weight is the weight of the backend in the routing decision.
 	Weight int `yaml:"weight"`
+	// Auth is the authn/z configuration for the backend. Optional.
+	// TODO: refactor after https://github.com/envoyproxy/ai-gateway/pull/43.
+	Auth *BackendAuth `yaml:"auth,omitempty"`
 }
 
+// BackendAuth ... TODO: refactor after https://github.com/envoyproxy/ai-gateway/pull/43.
+type BackendAuth struct {
+	AWSAuth *AWSAuth `yaml:"aws,omitempty"`
+}
+
+// AWSAuth ... TODO: refactor after https://github.com/envoyproxy/ai-gateway/pull/43.
+type AWSAuth struct{}
+
```

Comment:
Note: this will be refactored according to the final API of BackendSecurityPolicy in #43